main.go

```diff
@@ -31,28 +31,37 @@ func init() {
 func main() {
 	flag.Parse() // required for glog flags and testing package flags
 
-	v := viper.New()
-	config.SetupViper(v, "pbs")
-	cfg, err := config.New(v)
+	cfg, err := loadConfig()
 	if err != nil {
 		glog.Fatalf("Configuration could not be loaded or did not pass validation: %v", err)
 	}
-	if err := serve(Rev, cfg); err != nil {
+
+	err = serve(Rev, cfg)
+	if err != nil {
```

Comment:
The call to loadConfig cannot be inlined within the `if` block since the `cfg` is needed outside the block. Changing the usage of the server call to match for (hopefully you'll agree) improved readability.